airflow/dags/export_marts/dag.py

```diff
@@ -0,0 +1,58 @@
+import sqlalchemy
+import pandas as pd
+import boto3
+
+from io import StringIO
+from os import environ
+from airflow_operator import create_dag
+from airflow.decorators import dag,task
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+from airflow.hooks.subprocess import SubprocessHook
+from airflow.hooks.postgres_hook import PostgresHook
+from airflow.models import Variable
+
+
+
+dag = create_dag(
+    dag_id="export_dag",
+    schedule = "0 7 * * 2",
+    catchup=False,
+    concurrency=2
+)
+
+with dag:
+
+    @task
+    def export_tables():
+        pg_hook = PostgresHook(postgres_conn_id="postgres_default")
+        engine = pg_hook.get_sqlalchemy_engine()
+        marts_list = ["all_ndc_descriptions","atc_codes_to_rxnorm_products"]
+        mart_dfs={}
+        with engine.connect() as connection:
+            for mart in marts_list:
+                if sqlalchemy.inspect(engine).has_table(mart, schema='sagerx_dev'):  
```

Comment:
Should we put something here to determine if dev or prod environment and then either call sagerx schema or sagerx_dev respectively

Reply:
that would be interesting - I'm not sure how we would go about doing that.. I'm sure there's a way though